apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -1,10 +1,11 @@
 // Get router variables
+import autoAnimate from "@formkit/auto-animate";
 import { EventType } from "@prisma/client";
```

Comment:
Reading this file's diff is a lot easier if you click 'hide whitespace' and then 'apply and reload' at the top of this page:

<img width="419" alt="CleanShot 2022-10-05 at 15 25 37@2x" src="https://user-images.githubusercontent.com/2969573/194071658-2cab08a7-6fb9-4897-9e90-7a49728302ed.png">